### ISSUE TITLE
div style properties overrule inner span style properties

### ISSUE THREAD

**Issue by michvllni** [Aymkdn/html-to-pdfmake#49]
Hi again, sorry for coming up with another issue.

I'm using tinyMCE to generate the HTML code, and unfortunately, tinyMCE generates font-size properties even in the surrounding divs and even if they aren't even needed.
For both examples, I used the test site you provided.
So, when trying to convert this
```html
<div style="font-size: 7pt;"><span style="font-size: 11pt;">asda<strong>sd</strong>dad</span></div>
<div style="font-size: 7pt;">&nbsp;</div>
<div style="font-size: 7pt;"><span style="font-size: 11pt;">ublubub</span></div>
<div style="font-size: 7pt;">&nbsp;</div>
<div style="font-size: 7pt;"><span style="font-size: 11pt;">asdasd</span></div>
<div style="font-size: 7pt;">&nbsp;</div>
<div style="font-size: 7pt;"><span style="font-size: 11pt;">kl&ouml;asd<em>asd<u>asdasd</u></em></span></div>
<div style="font-size: 7pt;">&nbsp;</div>
```
I get the following result:
```json
[
  {
    "text": [
      {
        "text": "asda",
        "style": [
          "html-span"
        ]
      },
      {
        "text": "sd",
        "style": [
          "html-strong",
          "html-span"
        ],
        "bold": true
      },
      {
        "text": "dad",
        "style": [
          "html-span"
        ]
      }
    ],
    "style": [
      "html-span",
      "html-div"
    ],
    "fontSize": 7
  },
  {
    "text": " ",
    "fontSize": 7,
    "style": [
      "html-div"
    ]
  },
  {
    "text": "ublubub",
    "style": [
      "html-span",
      "html-div"
    ],
    "fontSize": 7
  },
  {
    "text": " ",
    "fontSize": 7,
    "style": [
      "html-div"
    ]
  },
  {
    "text": "asdasd",
    "style": [
      "html-span",
      "html-div"
    ],
    "fontSize": 7
  },
  {
    "text": " ",
    "fontSize": 7,
    "style": [
      "html-div"
    ]
  },
  {
    "text": [
      {
        "text": "klöasd",
        "style": [
          "html-span"
        ]
      },
      {
        "text": [
          {
            "text": "asd",
            "style": [
              "html-em",
              "html-span"
            ],
            "italics": true
          },
          {
            "text": "asdasd",
            "style": [
              "html-u",
              "html-em",
              "html-span"
            ],
            "decoration": "underline",
            "italics": true
          }
        ],
        "italics": true,
        "style": [
          "html-em",
          "html-span"
        ]
      }
    ],
    "style": [
      "html-span",
      "html-div"
    ],
    "fontSize": 7
  },
  {
    "text": " ",
    "fontSize": 7,
    "style": [
      "html-div"
    ]
  }
]
```
And it looks like this:
![image](https://user-images.githubusercontent.com/45660891/82025806-1cc0a400-9692-11ea-852f-cc723ce24726.png)

where the font size is 7 instead of the 11 of the span.

Additionally, `<br>` tags cause some weird behavior as well. It appears to variate the font-size in some lines, but not in all.

Example:
```html
<div style="font-family: segoe ui; font-size: 7pt;"><span style="font-size: 11pt;">asda<strong>sd</strong>dad</span><br></div>
<div style="font-family: segoe ui; font-size: 7pt;">&nbsp;<br></div>
<div style="font-family: segoe ui; font-size: 7pt;"><span style="font-size: 11pt;">ublubub</span><br></div>
<div style="font-family: segoe ui; font-size: 7pt;">&nbsp;<br></div>
<div style="font-family: segoe ui; font-size: 7pt;"><span style="font-size: 11pt;">asdasd</span><br></div>
<div style="font-family: segoe ui; font-size: 7pt;">&nbsp;<br></div>
<div style="font-family: segoe ui; font-size: 7pt;"><span style="font-size: 11pt;">kl&ouml;asd<em>asd<u>asdasd</u></em></span><br></div>
<div style="font-family: segoe ui; font-size: 7pt;">&nbsp;<br></div>
```
And the output:
```json
[
  {
    "text": [
      {
        "text": [
          {
            "text": "asda",
            "style": [
              "html-span"
            ]
          },
          {
            "text": "sd",
            "style": [
              "html-strong",
              "html-span"
            ],
            "bold": true
          },
          {
            "text": "dad",
            "style": [
              "html-span"
            ]
          }
        ],
        "style": [
          "html-span"
        ],
        "fontSize": 11
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": " "
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": "ublubub",
        "style": [
          "html-span"
        ],
        "fontSize": 11
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": " "
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": "asdasd",
        "style": [
          "html-span"
        ],
        "fontSize": 11
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": " "
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": [
          {
            "text": "klöasd",
            "style": [
              "html-span"
            ]
          },
          {
            "text": [
              {
                "text": "asd",
                "style": [
                  "html-em",
                  "html-span"
                ],
                "italics": true
              },
              {
                "text": "asdasd",
                "style": [
                  "html-u",
                  "html-em",
                  "html-span"
                ],
                "decoration": "underline",
                "italics": true
              }
            ],
            "italics": true,
            "style": [
              "html-em",
              "html-span"
            ]
          }
        ],
        "style": [
          "html-span"
        ],
        "fontSize": 11
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  },
  {
    "text": [
      {
        "text": " "
      },
      "\n"
    ],
    "style": [
      "html-div"
    ],
    "fontFamily": "segoeui",
    "fontSize": 7
  }
]
```
which then looks like this:
![image](https://user-images.githubusercontent.com/45660891/82025733-01559900-9692-11ea-95ae-a71c3deb0534.png)

**Comment by Aymkdn** [comment]
I've just published v1.3.0 that should resolve this issue.

**Comment by michvllni** [comment]
Thank you again for your fast fix :)
